rust/azure_iot_operations_services/src/azure_device_registry/client.rs

```diff
@@ -773,7 +777,9 @@ where
         timeout: Duration,
     ) -> Result<Asset, Error> {
         if asset_name.trim().is_empty() {
-            return Err(Error(ErrorKind::ValidationError("asset_name".to_string())));
+            return Err(Error(ErrorKind::ValidationError(
+                "{asset_name} is empty".to_string(),
```

Comment:
```suggestion
                "{asset_name} must not be empty".to_string(),
```
change for all, just to be clearer on how to fix the error

Reply:
oh wait, actually you'll want
>"asset_name must not be empty".to_string(),

because otherwise it will print `"" must not be empty` and not actually tell the issue